### DISCUSSION TITLE
联动中的run语句不生效

### DISCUSSION THREAD

**Discussion by cheng-xiaofeng** [alibaba/formily#3353] [Q&A 1.x] [answered]
我想在mapdata这个值改变的时候, 将其他的值清空, 但是这么写不生效, 为什么
![image](https://user-images.githubusercontent.com/33592850/185881074-d1dd632f-35b9-4601-a3f9-9756a1295bb5.png)

**Comment by janryWang** [comment, marked as answer]
不是这么玩的，你这个相当于受控改变schema，建议看看受控表单 文档

**Reply by cheng-xiaofeng** [reply]
我刚开始接触formily, 对这个不是很熟,你说的受控表单文档是哪个?能贴个地址吗

**Reply by janryWang** [reply]
https://formilyjs.org/zh-CN/guide/advanced/controlled
